Index: src/talk/media/webrtc/webrtcvoiceengine.cc
===================================================================
--- src/talk/media/webrtc/webrtcvoiceengine.cc	(revision 7524)
+++ src/talk/media/webrtc/webrtcvoiceengine.cc	(working copy)
@@ -750,7 +750,7 @@
   options.echo_cancellation.Set(false);
   options.auto_gain_control.Set(false);
 #elif defined(ANDROID)
-  ec_mode = webrtc::kEcAecm;
+  //ec_mode = webrtc::kEcAecm;
 #endif
 
 #if defined(IOS) || defined(ANDROID)
Index: src/webrtc/modules/audio_device/android/opensles_input.h
===================================================================
--- src/webrtc/modules/audio_device/android/opensles_input.h	(revision 7524)
+++ src/webrtc/modules/audio_device/android/opensles_input.h	(working copy)
@@ -124,7 +124,7 @@
     // Keep as few OpenSL buffers as possible to avoid wasting memory. 2 is
     // minimum for playout. Keep 2 for recording as well.
     kNumOpenSlBuffers = 2,
-    kNum10MsToBuffer = 3,
+    kNum10MsToBuffer = 10,
   };
 
   int InitSampleRate();
Index: src/webrtc/modules/audio_device/android/opensles_output.h
===================================================================
--- src/webrtc/modules/audio_device/android/opensles_output.h	(revision 7524)
+++ src/webrtc/modules/audio_device/android/opensles_output.h	(working copy)
@@ -140,7 +140,7 @@
     // there will be jitter in audio pipe line due to the acquisition of locks.
     // Note: The buffers in the OpenSL queue do not count towards the 10ms of
     // frames needed since OpenSL needs to have them ready for playout.
-    kNum10MsToBuffer = 6,
+    kNum10MsToBuffer = 10,
   };
 
   bool InitSampleRate();
